### DISCUSSION TITLE
Update the Raspbian installation documentation

### DISCUSSION THREAD

**Discussion by ThierryM** [ltsp/ltsp#328] [Show and tell] [answered]
Hi,
Thanks for the sharing.
I'm intenting to use LSTP 20 under an Ubuntu 20.04 server with Raspberry Pi 4 clients. I saw the useful explanations here : https://ltsp.org/docs/installation/raspbian/.
The command lines with the "echo" function don't work (permission denied). So instead this command line : 
`echo 'proc            /proc           proc    defaults          0       0' >./etc/fstab`
I had to use this line with sudo tee -a : 
`echo 'proc            /proc           proc    defaults          0       0' |  sudo tee -a ./etc/fstab > /dev/null`
So I think that the lines with echo should be changed.

I'm writing a page to explain in french how to use LTSP 20 for Raspberry Pi 4 and the french educationnal distribution Primtux (based on Raspberry Pi OS) here : https://lofurol.fr/joomla/electronique/raspberry-pi/263-utiliser-des-raspberry-pi-4-en-tant-que-clients-lourds

Regards,

Thierry

**Comment by alkisg** [comment]
Hi, all LTSP commands should be run as root, i.e. you're supposed to use `sudo -i` or `su -` before running them.

**Reply by ThierryM** [reply]
Yes, I use sudo all the time but even this line :
`sudo echo 'proc /proc proc defaults 0 0' >./etc/fstab`
doesn't work.

**Comment by ThierryM** [comment]
An other problem I had, it's for the chrooting preparation. I had to add the option "norecovery' in order to mount the image.
So this line doesn't work : 
`sudo mount -o ro /dev/loop8p2 /mnt` 
But with norecovery, no problem : 
`sudo mount -o ro,norecovery /dev/loop8p2 /mnt`

**Comment by alkisg** [comment, marked as answer]
> Yes, I use sudo all the time but even this line doesn't work

Yes, this is not the correct way to use sudo. As I said, please use `sudo -i` to become root, and THEN run the LTSP commands that are mentioned in the manual.

```shell
sudo -i
ltsp image, echo, other commands go in the following lines
```

>  I had to add the option "norecovery'

This means that your image wasn't unmounted properly, it's not an issue with the documentation either.

**Reply by ThierryM** [reply]
Thanks for yours precisions. I'll give a try and I report here.
Regards.

**Reply by ThierryM** [reply]
I found my errors : 
First the sudo -i instead of using sudo before all the command lines.
Two, I didn't wait sufficiently for mounting the .img : some seconds.
Thank you very much for your help.
Regards

**Reply by ThierryM** [reply]
The mounting problem that I had was caused by a lack of place in my hard disk... Silly error... as often.
